dolosse/generic_json_consumer.py

```diff
@@ -0,0 +1,153 @@
+"""Reads json data from a kafka topic and graphs it"""
+
+from argparse import ArgumentParser
+from curses import initscr, cbreak, ERR, nocbreak, endwin
+from json import loads, decoder
+
+from confluent_kafka import Consumer
+from matplotlib import pyplot
+
+
+def populate(variables, decode,
+             count, prepend=""):
+    """Takes the data from the dict and populates the variables"""
```

Comment:
Missing parameter and return documentation.

Reply:
Added.